assign-pull-requests.py

```diff
@@ -126,6 +127,25 @@ def main(ref_repo_path):
     return 0
 
 
+def commit_contains_correct_signoff(commit):
+    """ Given a commit, it is considered to contain a correct sign-off IFF:
+    It has a footer line starting with 'Signed-off-by: '
+    AND that line contains the email address of the committer.
+
+    Other sign-off lines MAY also be present, but do not affect the outcome of
+    this check. It's entirely valid for one person to have multiple sign-off
+    lines, using different email addresses (signing off on something for both
+    work and Gentoo development)
+    """
+    committer_email = commit.committer.email.lower()  # case insensitive
+    for line in commit.message.splitlines():
+        if line.lower().startswith('signed-off-by:'):
```

Comment:
I suppose you could store `line.lower()` in a helper var, and avoid the `.lower()` below too.